torch/_inductor/config.py

```diff
@@ -175,6 +175,9 @@ class triton:
     # should we put op names in kernel names
     descriptive_kernel_names = False
 
+    # use alternate codegen for smaller reductions
+    persistent_reductions = False
```

Comment:
Should this be on by default?

Reply:
Going to leave this off as this is triggering some rare triton errors.